hknweb/candidate/static/candidate/style.css

```diff
@@ -204,16 +204,46 @@ h2:first-child {
     color: #0000A0;
 }
 
-.gray-note {
-    color: #727c81;
-    font-size: 0.8em
-}
-
 .table-icon {
     width: 1.5em;
     vertical-align: middle;
 }
 
+/* mouseover info for forms */
+.tooltip {
+    position: relative;
+    display: inline-block;
+}
+
+.tooltip .tooltiptext {
+    visibility: hidden;
+    width: 1350%;
+    background-color: #f2f2f2;
+    color: black;
+    font-size: 0.8em;
+    padding: 0.4em;
+    border-style: groove;
+    border-radius: 8px;
+    position: absolute;
+    top: -40%;
+    left: 150%;
+}
+
+.tooltip .tooltiptext::after {
+    content: "";
+    position: absolute;
+    top: 15%;
+    right: 101.2%;
```

Comment:
This seems awfully specific, what is this for?

Reply:
I think it was for the small arrow coming out of the text box, it needs to be exactly on the box. These numbers don't work for all zoom levels but they're mostly good

Reply:
I added some comments